x/gov/abci.go

```diff
@@ -72,6 +72,85 @@ func EndBlocker(ctx sdk.Context, keeper *keeper.Keeper) error {
 		return err
 	}
 
+	// fetch active proposals that need to be checked for quorum
+	rng = collections.NewPrefixUntilPairRange[time.Time, uint64](ctx.BlockTime())
+	err = keeper.QuorumCheckQueue.Walk(ctx, rng, func(key collections.Pair[time.Time, uint64], quorumChecksDone uint64) (bool, error) {
+		proposal, err := keeper.Proposals.Get(ctx, key.K2())
+		if err != nil {
+			return false, err
+		}
+
+		params, err := keeper.Params.Get(ctx)
+		if err != nil {
+			return false, err
+		}
+
+		// remove from queue
+		err = keeper.QuorumCheckQueue.Remove(ctx, key)
+		if err != nil {
+			return false, err
+		}
+
+		if params.QuorumCheckCount == 0 {
+			return false, nil
+		}
+
+		proposalVotingPeriod := proposal.VotingEndTime.Sub(*proposal.VotingStartTime)
+		s := proposalVotingPeriod - *params.QuorumTimeout
```

Comment:
`s` is too short for a variable name used 30 lines below. Let's call it `QuorumCheckPeriod` or something similar.

Reply:
It is admissible and code etiquette in Go to use short names for variables, but you are right

Reply:
https://github.com/giunatale/cosmos-sdk/commit/6c5ca356b88ef382dbf0d32b85a06b1d80bd1738 addressed, but I checked and still can't push to this repository

Reply:
I pushed your change